build/helpers/file-includes-all-subdirectories-as-named-exports.spec.ts

```diff
@@ -0,0 +1,20 @@
+import Adapter from 'enzyme-adapter-react-16'
+import { configure } from "enzyme";
+import * as path from "path";
+import { includesAllSubdirectoriesAsNamedExports } from "./file-includes-all-subdirectories-as-named-exports";
+
```

Comment:
I don't think you really need any of this enzyme setup to test exports.

Reply:
Removing all enzyme stuff.